playwright/register-form-validation.spec.ts

```diff
@@ -22,47 +55,41 @@ async function fillInitialsInput(page: Page, user: User): Promise<void> {
 
 async function fillEmailInput(page: Page, user: User): Promise<void> {
   const emailInput: Locator = page.getByPlaceholder('vilnaCRM@gmail.com');
-  await emailInput.fill('hello@sdf');
-  await expect(
-    page.getByText("Email must contain '@' and '.' symbols")
-  ).toBeVisible();
+  for (const expectation of expectations) {
+    await emailInput.fill(expectation.email);
+    await expect(page.getByText(expectation.errorText)).toBeVisible();
+  }
 
-  await emailInput.fill('hello@sdf.');
-  await expect(page.getByText('Invalid email address')).toBeVisible();
   await emailInput.fill(user.email);
 }
 
 async function fillPasswordInput(page: Page, user: User): Promise<void> {
   const passwordInput: Locator = page.getByPlaceholder('Create a password');
-  await passwordInput.fill('tirion');
-  await expect(page.getByText('Password must be between 8')).toBeVisible();
-
-  await passwordInput.fill('lanister');
-  await expect(
-    page.getByText('Password must contain at least one number')
-  ).toBeVisible();
-
-  await passwordInput.fill('lanister1');
-  await expect(
-    page.getByText('Password must contain at least one uppercase letter')
-  ).toBeVisible();
+  for (const expectation of expectationsPassword) {
```

Comment:
same

Reply:
It was a comfortable way for me, should I change it?